crmsh/utils.py

```diff
@@ -2198,4 +2198,59 @@ def check_ssh_passwd_need(hosts):
         if rc != 0:
             return True
     return False
+
+
+def check_port_open(host, port):
+    import socket
+
+    with closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as sock:
+        if sock.connect_ex((host, port)) == 0:
+            return True
+        else:
+            return False
+
+
+def valid_port(port):
+    return int(port) >= 1024 and int(port) <= 65535
+
+
+def use_qdevice():
+    from . import corosync
+    return corosync.get_value("quorum.device.model") == "net"
+
+
+def qdevice_tls_on():
```

Comment:
The same, maybe change to `is_qdevice_tls_on`

Reply:
changed